Description: fix convert atomic to non-atomic integer for UniValue
 Fixes Bitcoin 0.20.1 build failure when linked with mainline UniValue:
 rpc/blockchain.cpp:2077:35: error: cannot convert 'std::atomic<int>' to 'const UniValue&'
Author: Jonas Smedegaard <dr@jones.dk>
License: Expat
Last-Update: 2020-11-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/rpc/blockchain.cpp
+++ b/src/rpc/blockchain.cpp
@@ -2187,7 +2187,7 @@
             // no scan in progress
             return NullUniValue;
         }
-        result.pushKV("progress", g_scan_progress);
+        result.pushKV("progress", (int)g_scan_progress);
         return result;
     } else if (request.params[0].get_str() == "abort") {
         CoinsViewScanReserver reserver;
